autoload/asyncomplete.vim

```diff
@@ -1,3 +1,15 @@
+if !has('timers')
+    echohl ErrorMsg
+    echomsg 'Vim/Neovim compiled with timers required for asyncomplete.vim.'
+    echohl NONE
+    if has('nvim')
+        call asyncomplete#log('neovim compiled with timers required.')
```

Comment:
Also I think this line should be changed to:

> Newer version of NeoVim is required. This doesn't support timers.

As NeoVim doesn't have "compilation flags" to enable or disable features. So only way that someone doesn't have it when using version earlier than 0.1.5.